symphony-ai/src/main/java/org/symphonyoss/symphony/bots/ai/impl/AiImpl.java

```diff
@@ -10,6 +10,8 @@
 import org.symphonyoss.symphony.bots.ai.model.AiSessionKey;
 
 /**
+ * Concrete implementation of {@link Ai}
```

Comment:
What is AI?

Reply:
A description was add in this class interface, in the SymphonyAI class and in the pom's project description